safehttp/mux.go

```diff
@@ -92,35 +89,41 @@ func NewServeMux(d Dispatcher, domains ...string) *ServeMux {
 		dm[host] = true
 	}
 	return &ServeMux{
-		mux:       http.NewServeMux(),
-		domains:   dm,
-		disp:      d,
-		handlers:  map[string]methodHandler{},
-		interceps: map[string]Interceptor{},
+		mux:      http.NewServeMux(),
+		domains:  dm,
+		disp:     d,
+		handlers: map[string]methodHandler{},
 	}
 }
 
+type appliedInterceptor struct {
```

Comment:
Maybe `configuredInterceptor` or so? Also, add a brief comment to explain what this is (I know it's unexported but I think it would help people maintaining this package in the future).

Reply:
What needs to be explained in the comment? This is just a holder of data, it has no methods. There is no depth that needs to be explained. I think a comment would just add clutter in this case.